prac5/word_occurrences.py

```diff
@@ -0,0 +1,13 @@
+word_to_count = {}
+user_text = input('text')
+words = user_text.split()
+longest_word_length = 0
+for word in words:
+    if word not in word_to_count:
+        word_to_count[word] = 1
+    else:
+        word_to_count[word] += 1
```

Comment:
A return (enter) may help improve readability of the program

Reply:
thanks read ability improved